packages/cashier/src/stores/general-store.ts

```diff
@@ -138,13 +134,16 @@ export default class GeneralStore extends BaseStore {
     showP2pInCashierOnboarding(): void {
         const { account_list, is_virtual } = this.root_store.client;
 
-        const is_p2p_restricted = this.p2p_advertiser_error === 'RestrictedCountry';
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const website_status: any = this.WS.wait('website_status');
```

Comment:
You can use `ServerStatusResponse` from `@deriv/api-types` instead of `any`, But would be better to type it in `packages/cashier/src/types/websocket.types.ts` 🙇🏻

Reply:
ahhh thanks for letting me know!